packages/retail-ui/components/TopBar/TopBar.less

```
@@ -91,7 +91,6 @@
   }
 
   .iconOnly {
-    line-height: 54px;
     padding: 0 11px;
   }
```

Не стоит ли тогда убрать `margin-right` у иконки?

```suggestion
    .icon {
      margin-right: 0;
    }
  }
```

---

Тогда между текстом и иконкой расстояние будет слишком маленьким. Проще не устанавливать класс `icon`, если есть проп `iconOnly`

---

Ну я почти тоже самое сделал. Если `iconOnly` не указан, то добавляю иконке margin